toqito/perms/symmetric_projection.py

```diff
@@ -87,7 +88,7 @@ def symmetric_projection(
         return np.eye(dim)
 
     p_list = np.array(list(permutations(np.arange(1, p_val + 1))))
-    p_fac = np.math.factorial(p_val)
+    p_fac = math.factorial(p_val)
```

Comment:
I'm curious why we need to `import math` and use the `math` factorial over the `numpy` one:
https://stackoverflow.com/questions/21753841/factorial-in-numpy-and-scipy

Reply:
Due to a deprecation warning. 

```
 DeprecationWarning: `np.math` is a deprecated alias for the standard library `math` module (Deprecated Numpy 1.25). Replace usages of `np.math` with `math`
    p_fac = np.math.factorial(p_val)

```

Reply:
Ah, true `np.math` is deprecated, but is `np.factorial`? I think *that* would be okay to use, right?

Reply:
No. ` AttributeError: module 'numpy' has no attribute 'factorial'`

Reply:
🤷 `math.factorial` it is! :)